django/curator/management/commands/curator_spam_detection.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import pathlib
+
+from django.core.management.base import BaseCommand
+
+from curator.spam import DATASET_FILE_PATH
+from curator.spam import SpamDetection
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Perform spam detection"
+
+    def __init__(self):
+        self.detection = SpamDetection()
+        self.processor = self.detection.processor
+        self.user_meta_classifier = self.detection.user_meta_classifier
+        self.text_classifier = self.detection.text_classifier
+
```

Comment:
I think this may be better done with subcommands/subparsers (https://docs.python.org/3/library/argparse.html#sub-commands) since it seems the intent is to just run one action. Also may want to consider combining common workflows

Reply:
I'm still trying to figure out what to fix for this suggestion. I'll ask you again today at our meeting.